src/core/cells.cpp

```diff
@@ -68,6 +68,15 @@ double max_range = 0.0;
 unsigned resort_particles = Cells::RESORT_NONE;
 int rebuild_verletlist = 1;
 
+
+ParticleRange CellStructure::get_local_particles() const {
```

Comment:
Please add instead methods that return the cell lists, and get the particles from them. Also I am
not in favor of the `get_`, the verb suggests an action, but this only refers to an existing entity.